server/athenian/api/openapi/openapi.yaml

```diff
@@ -1314,9 +1314,7 @@ components:
         values:
           description: Calculated metric values in the same order as `metrics`.
           items:
-# {% if False %}
-            allOf:
-# {% else %}{{ "\n" }}            oneOf:{% endif %}
+            oneOf:
```

Comment:
Is this fix no longer required?

Reply:
It is **required**.

I just created the PR from the wrong branch, so you saw here what should be at https://github.com/dpordomingo/athenian-api/pull/4 in order to provide an example of the failing case. Sorry for letting you see an incomplete PR.